apps/docs/customization/proxy/caching.md

```
@@ -0,0 +1,65 @@
+---
+title: Proxy — Host caching
+description: Host-resolution caching for @authhero/proxy — in-memory SWR and pluggable CacheAdapter (Cloudflare Cache API, Redis) for cross-isolate hits.
+---
+
+# Host cache
+
+`resolveHost` runs on every request. Two cache implementations ship:
+
+### In-memory (default)
```

markdownlint-cli2 (0.22.1)

[warning] Heading levels should only increment by one level at a time
Expected: h2; Actual: h3

(MD001, heading-increment)